src/Shared/Lib.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace aoc.Shared
+{
+	public static class Lib
+	{
+		public static List<List<string>> GetEmptyLineSeparatedChunks(string [] lines)
+		{
+			var chunks = new List<List<string>>();
+			var chunk = new List<string>();		
+
+			for (int i = 0; i < lines.Length; i++)
```

Comment:
why not a `foreach`

Reply:
@Jaxwood I need special processing at last line. How do you check if its last line in a  foreach?